dhall/dhall.cabal

```diff
@@ -26,6 +26,7 @@ Extra-Source-Files:
     benchmark/deep-nested-large-record/*.dhall
     benchmark/examples/*.dhall
     benchmark/examples/normalize/*.dhall
+    benchmark/normalize/*.dhall
```

Comment:
Please move that file to `benchmark/examples`.

Reply:
Sure!